### ISSUE TITLE
Hyperparameter settings with MobileNet

### ISSUE THREAD

**Issue by BJTUJia** [KaiyangZhou/deep-person-reid#7]
Dear Author, Thanks a lot for your excellent work. If I may, can I ask you to provide the details of your hyperparameter settings? For example, I am training MobileNet V2 on Market, when I use the default settings, such as lr=0.0003, the training can only get 11.9% accuracy. I use lr=0.01 and decrease by x0.1 every 20 epoches, the training accuracy can hit 98.%, but the test performance is 68.1%, 85.5%,90.4%(rank 1 5 10) and map is 44.6%, which is much lower than your results. Can you point out what's wrong? FYI: I did't use crossentropy with label smooth, I use standard cross-entropy. will this have an impact on the influence?

**Comment by KaiyangZhou** [comment]
MobileNetV2 is trained from scratch, so more epochs are required for better convergence. You can try `--max-epoch 300 --stepsize 100 --eval-step 50` and keep other settings unchanged. Label smoothing is critical as Market1501 does not contain (diverse) enough images for training CNN with millions of params.

**Comment by simon-rob** [comment]
I am having the same issue with MobilenetV2.

@KaiyangZhou  you mentioned that label smoothing is important. So could the problem be:

``
 if args.label_smooth:
        criterion = CrossEntropyLabelSmooth(num_classes=dataset.num_train_pids, use_gpu=use_gpu)
    else:
        criterion = nn.CrossEntropyLoss()
``

isn't being enabled, because: --**label-smooth** should be **label_smooth** on line 78? 

``
parser.add_argument('--label-smooth', action='store_true',
                    help="use label smoothing regularizer in cross entropy loss")
``

Also what optimizer and scheduler type did you use to train mobilenet V2: adam?

**Comment by d-li14** [comment]
@BJTUJia Hi, how about the final performance you have achieved after tuning the parameters?
